chain/client/src/stateless_validation/chunk_validator/orphan_witness_handling.rs

```diff
@@ -0,0 +1,199 @@
+//! This module contains the logic of handling orphaned chunk state witnesses.
```

Comment:
nit: I thing `orphan_witness.rs` is descriptive enough, we are not in java world 😉

Reply:
I like descriptive names c:
`orphan_witness` doesn't say much about what happens inside, but `orphan_witness_handling` clearly states that this module contains code which handles orphan witnesses.